src/Dao/EnwikiDao.php

```diff
@@ -47,6 +47,29 @@ public function __construct( $dsn, $user, $pass, $wiki = 'https://en.wikipedia.o
 
 
 	/**
+	 * Get details on multiple revisions
+	 *
+	 * @param $diffs array Revision IDs
+	 * @return array full revision rev_id, rev_user, rev_usertext, user_editcount and user_name of the revision
+	 */
+	public function getRevisionDetailsMulti( $diffs ) {
+		$query = self::concat(
+			'SELECT r.rev_id, r.rev_user, r.rev_user_text, u.user_editcount, u.user_name',
+			'FROM revision r',
+			'LEFT JOIN user u ON r.rev_user = u.user_id',
+			'WHERE r.rev_id IN (' . join( ',', $diffs ) . ')' // FIXME: could not get prepared statement (?) to work
```

Comment:
To do this as a prepared statement you will need to insert a placeholder for each value in $diffs. Something like:

```
'WHERE r.rev_id IN (' . implode( ',', array_fill( 0, count( $diffs ), '?' ) ) . ')'
```
Then you can pass `$diffs` directly as the parameter collection to `fetchAll()`.